### ISSUE TITLE
Brand request: TruthSocial

### ISSUE THREAD

**Issue by TylerMWise** [FortAwesome/Font-Awesome#19367]
### Description

Hi there,
I am requesting a TruthSocial brand icon for the social media platform [TruthSocial](https://truthsocial.com/).

Proposed class name: fa-truthsocial

![image](https://user-images.githubusercontent.com/74430503/194645338-3d8c49e7-247e-439c-a4ba-826a9581495f.png)

![image](https://user-images.githubusercontent.com/74430503/194645358-779fb540-57c8-4960-ba27-89f7d78643bf.png)

### Official website

https://truthsocial.com/

### Icon request checklist

- [X] The title starts with "Brand request: ", is followed by the requested brand name
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/brands#faqs)

**Comment by TylerMWise** [comment]
1+

**Comment by emkowale** [comment]
I would like a TruthSocial icon as well. It is a growing and widely used platform. Please keep political bias out of github.

**Comment by NicholasEli** [comment]
Bump

**Comment by dgsinclair** [comment]
Bump

**Comment by clarkdowding** [comment]
Client wants this icon on her profile website but Font Awesome doesn't have it?

**Comment by frisch1** [comment]
**Bump**

**TL;DR**
My general disdain for partisan anything is fighting with my overwhelming laziness, nurtured in part by FontAwesome the last decade having every icon I could ever need before I think I need it. So really, this Bump is _all your fault, FontAwesome_. ;)

And I was going to say "And we'll probably need Bluesky soon" but luckily I checked first before embarrassing myself...

**Comment by frisch1** [comment]
Joking aside, if helpful, here's the SVG:
```
<svg viewBox="0 0 267 214" xmlns="http://www.w3.org/2000/svg">
	<path d="M 267 156.3 L 197.164062 156.3 L 197.164062 213.390625 L 267 213.390625 Z"/>
	<path d="M 239.6875 57.636719 L 239.6875 0 L 97.386719 0 L 97.386719 213.273438 L 170.019531 213.273438 L 170.019531 57.636719 Z"/>
	<path d="M 0 0 L 69.726562 0 L 69.726562 57.644531 L 0 57.644531 Z"/>
</svg>
```

[Download](https://media-cdn.factba.se/icons/truth_social.svg)

**Comment by tylerdisney** [comment]
Can we please get this icon added to the brands set? It's outrageous that BlueSky was added within days, but Truth still isn't there years later.

I understand the disdain for the brand in general, but leaving it out of your library doesn't hurt the people/clients that want to use it. It only makes it more inconvenient for us devs (your actual customers) that are asked to use the brand, and instead have to take more time to place SVG versions up on client websites and match them to the rest of your library. I don't care about politics, I just want a usable COMPREHENSIVE icon library. 

Thank You.

**Comment by 42kohd** [comment]
Makes fontawesome look stupid not to have this icon. Clients want it.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
